charts/gardener/provider-local/internal/shoot-system-components/charts/local-path-provisioner/templates/psp.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if not .Values.pspDisabled }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: gardener.kube-system.local-path-provisioner
+  annotations:
+    seccomp.security.alpha.kubernetes.io/defaultProfileName: 'runtime/default'
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'runtime/default'
+spec:
+  privileged: true
+  allowPrivilegeEscalation: true
+  allowedCapabilities:
+  - '*'
+  volumes:
+  - '*'
+  hostNetwork: true
+  hostPorts:
+  - min: 0
+    max: 65535
+  hostIPC: true
+  hostPID: true
+  runAsUser:
+    rule: 'RunAsAny'
+  seLinux:
+    rule: 'RunAsAny'
+  supplementalGroups:
+    rule: 'RunAsAny'
+  fsGroup:
+    rule: 'RunAsAny'
```

Comment:
Do we need these much privileges to solve the issue in hand?

Reply:
It is required in local setup, so there is no harm in giving all access.